zephyr/docker-build.sh

```diff
@@ -71,8 +71,4 @@ unset ZEPHYR_SDK_INSTALL_DIR
 ls -ld /opt/toolchains/zephyr-sdk-*
 ln -s  /opt/toolchains/zephyr-sdk-*  ~/ || true
 
-if test -e zephyrproject; then
-    ./scripts/xtensa-build-zephyr.py  "$@"
-else # -c(lone) with west init etc.
-    ./scripts/xtensa-build-zephyr.py -c "$@"
-fi
+./sof/scripts/xtensa-build-zephyr.py -u --no-interactive "$@"
```

Comment:
I added this `if/then/else` so this docker script can be tested locally, outside CI without having to delete and re-clone everything.

Now things are different and it's not such a problem anymore but I think an `if/then/else` would still be nice to drop the `-u` when everything is already cloned, it would save a bit of time. No big deal, can be re-added later.